assets/js/theme/blocks/keyFigures.js

```diff
@@ -15,9 +15,10 @@ class KeyFigures {
     constructor (dom) {
         this.dom = dom;
         this.time = 0;
+        this.isAnimated = this.dom.classList.contains('is-animated') ? true : false;
```

Comment:
Tu n'as pas besoin de la condition ternaire ici car contains() renvoie un booléen déjà 

```suggestion
        this.isAnimated = this.dom.classList.contains('is-animated');
```

Reply:
ah mais oui, merci !